library/physics.py

```diff
@@ -68,3 +68,29 @@ def get_angular_velocity(self) -> NDArray[3, np.float32]:
             yaw = ang_vel[1]
         """
         pass
+
+    @abc.abstractmethod
+    def get_position(self) -> NDArray[3, np.float32]:
+        """
+        Returns a 3D vector containing the car's position as x, y, and z coordinates.
+
+        Returns:
+            The position of the car along the (x, y, z) axes during
+            the last frame in m.
+
+        Note:
+            The x axis points out of the right of the car.
```

Comment:
This note is not relevant because the xyz axes used by the GPS are relative to the world, not the car. If the car rotates in place, it's GPS coordinates do not change.

Instead, give any relevant notes about what the *world* axes correspond to. Which one is North, which one is East, and which one is altitude?